src/qml/README.md

```diff
@@ -21,52 +21,67 @@ The primary goals of the project can be summed up as follows:
+aim to make very few changes outside of the qml directory.
 Pull requests must be focused on developing the GUI itself, adding build support,
 or improving relevant documentation.
 
-Note that this project will **not** accept pull requests making any significant changes unrelated to the GUI.
+This project will **not** accept pull requests making any significant changes unrelated to the GUI.
 
-# Development Process
+## Development Process
 
 This repo is synced with the [Bitcoin Core repo](https://github.com/bitcoin/bitcoin) on a weekly basis, or as needed to resolve conflicts.
 
 Contributions are welcome from all, developers and designers. If you are a new contributor, please read [CONTRIBUTING.md](../../CONTRIBUTING.md).
 
-## Minimum Required Qt Version
+### Minimum Required Qt Version
 
 All development must adhere to the current upstream Qt Version to minimize our divergence from upstream and avoid costly changes. Review of open PR's must ensure that changes are compatible with this Qt version. Currently, the required version is [Qt 5.12.11](https://github.com/bitcoin-core/gui-qml/blob/a79a2249d027047f08dffe3f04951d52eac198f1/depends/packages/qt.mk#L2).
 
 As the Qt Version changes upstream, refactoring is allowed to use the now available features.
 
-# Compile and Run
+### Policies
 
-The master branch is only guaranteed to work and build on Debian-based systems and macOS.
+This project has custom policies for development, see:
+  - [Icon Policy](./doc/icon-policy.md)
+  - [Translator Comments Policy](./doc/translator-comments.md)
+
+## Compile and Run
+
+The master branch is only guaranteed to work and build on Debian-based systems, Fedora, and macOS.
 Support for more systems will be confirmed and documented as the project matures.
 
 ### Dependencies
-Aside from the dependencies listed in [build-unix.md](../../doc/build-unix.md), Debian based systems require the following additional dependencies to compile:
+No additional dependencies, besides those in [build-osx.md](../../doc/build-osx.md), are needed for macOS.
+
+Aside from the dependencies listed in [build-unix.md](../../doc/build-unix.md), the following additional dependencies are required to compile:
+
+#### Debian-based systems:
 
 ```
 sudo apt install qtdeclarative5-dev qtquickcontrols2-5-dev
 ```
 
-The following runtime dependencies are also required for dynamic builds;
-they are not needed for static builds:
+  The following runtime dependencies are also required for dynamic builds;
+  they are not needed for static builds:
 
 ```
 sudo apt install qml-module-qtquick2 qml-module-qtquick-controls2 qml-module-qtquick-layouts qml-module-qtquick-window2
 ```
 
-No additional dependencies, besides those in [build-osx.md](../../doc/build-osx.md), are needed for macOS.
+#### Fedora:
+
+```
+sudo dnf install qt5-qtdeclarative-devel qt5-qtquickcontrols2-devel
+```
 
 ### Build
 
 For instructions on how to build and compile Bitcoin Core, refer to your respective systems build docs.
```

Comment:
Since we are improving the docs, I had one suggestion regarding a line not tackled in this PR.

**Suggestion:**
```suggestion
For instructions on building and compiling Bitcoin Core, refer to your respective system's build-docs under the `doc/build-*.md` file.
```
Changes:
- **Explicitly specifying where to find build docs**. I think this would be helpful for someone new to the bitcoin-core.
- **Respective grammatical changes**. These improves the flow and ease of read for the sentence.

Reply:
Thanks for the review! I like the current wording of 'how to build'. I did make the following change though:

```suggestion
For instructions on how to build and compile Bitcoin Core, refer to your respective system's build doc.
```